docs/manual-EN/3.build-develop-and-administration/4.account-management-statements/authentication.md

```diff
@@ -0,0 +1,49 @@
+
+## Local Authentication
+
+The local database stores usernames, encrypted passwords, local user settings and remote LDAP user settings. When a user tries to access the database, they will be met with a security challenge.
+
+You need to enable the local authentication by setting the `--enable_authorize` property in the `nebula-graphd.conf` file (the directory is `/usr/local/nebula/etc/` by default) to `true`.
+
+## LDAP Authentication
+
+Lightweight Directory Access Protocol (LDAP) is a lightweight client-server protocol for accessing directory services. Users stored inside LDAP take precedence over the local database users. For example, if both providers have a user called “Amber”, the settings and roles for this user will be sourced from LDAP.
+
+Unlike local authentication, besides enabling the `--enable_authorize` parameter, LDAP needs to be configured in the `nebula-graphd.conf` file (the directory is `/usr/local/nebula/etc/` by default). Refer to the [Integrating LDAP Document](TODO) for details.
+
+When LDAP is turned on, the following security settings can be used to configure it:
+
+### ObjectClass
+
+ObjectClass attribute specifies the object classes of an entry, which (among other things) are used in conjunction with the controlling schema to determine the permitted attributes of an entry.
+
+| objectClass | Description |
+| --- | --- |
+| olcGlobal | Type of the global configuration file. `cn=config.ldif` |
+| top | Top objects |
+| organization | Organization may be Organizational Entity like a Company or an Employer. |
+| organizationalUnit | Generally assumed to be a relatively static grouping within a larger organization.  |
+| inetOrgPerson | Real user type.  In Active Directory, this class has user as a parent class. |
+| groupOfNames | A LDAP group. |
+| olcModuleList | Object of the configured module list. |
+
+<!-- ### LDAP Keywords
+
+| Keywords | Full Name | Description |
+| --- | --- | --- |
+| dc | Domain Component | The dc refers to each component of the domain. For example example.com can be written as dc=example,dc=com |
+| uid | User Id | uid is an LDAP attribute that means user id. For example “tom”. |
+| ou | Organization Unit | The organizational unit attribute refers to the organizational unit (or sometimes the user group) that the user is part of.|
+| cn | Common Name | Common Names. For example “Thomas Johnson”. |
+| sn | Surname | Surnames. For example “Johnson”. |
+| dn | Distinguished Name | The unique identifier for an entry in the tree, similar to the absolute path in the Linux file system. For example “uid= tom,ou=market,dc=example,dc=com” is unique in a tree.|
+| rdn | Relative dn | Each entry has a unique name relative to its parent called RDN. For example “uid=tom” or “cn= Thomas Johnson”. |
+| c | Country | c is an LDAP attribute that means country name. For example "CN" and "US".|
+| o | Organization | Organization name. For example “Example, Inc.” | -->
```

Comment:
line23 to line 49, What do I not understand?
I think it should be deleted.

Add parameter interpretation:
```
// LDAP authentication common parameters
DEFINE_string(ldap_server, "", "list of ldap server addresses, "
                               "separate multiple addresses with comma");
DEFINE_int32(ldap_port, 0, "Ldap server port, if no port is specified, use the default port");
DEFINE_string(ldap_scheme, "ldap", "Only support ldap");
DEFINE_bool(ldap_tls, false, "Whether the connection between graphd and the LDAP server uses "
                             "TLS encryption");

// LDAP authentication simple bind mode parameters
DEFINE_string(ldap_prefix, "", "Prepend the string to the user name "
                               "to form the distinguished name");
DEFINE_string(ldap_suffix, "", "Append the string to the user name "
                               "to form the distinguished name");

// LDAP authentication search bind mode parameters
DEFINE_string(ldap_basedn, "", "Root distinguished name to search the user");
DEFINE_string(ldap_binddn, "", "User distinguished name binding to the "
                               "directory to perform the search");
DEFINE_string(ldap_bindpasswd, "", "User password binding to the directory "
                                   "to perform the search");
DEFINE_string(ldap_searchattribute, "", "Attribute to match the user name in the search");
DEFINE_string(ldap_searchfilter, "", "Use search filter, more flexible than searchattribut");
```